blimflix/client/app/directives/side-bar/side-bar.directive.js

```diff
@@ -1,56 +1,24 @@
 (function () {
   'use strict';
 
-  /**
-   * Chain to fetch module
-   *   https://github.com/johnpapa/angular-styleguide#style-y022
-   */
-
-  /**
-   * Create one directive per file. Name the file for the directive.
-   *   https://github.com/johnpapa/angular-styleguide#style-y070
-   */
-
   angular
     .module('blimFlix.directives')
     .directive('sideBar', sideBar);
 
-  /**
-   * Avoid anonymous functions as callbacks
-   *   https://github.com/johnpapa/angular-styleguide#style-y024
-   */
   function sideBar() {
-    /**
-     * When creating a directive that makes sense as a stand-alone element, allow restrict E (custom element) and optionally restrict A (custom attribute).
-     * EA is the default for Angular 1.3 +
-     *   https://github.com/johnpapa/angular-styleguide#style-y074
-     */
-
-    /**
-     * Use "controllerAs" syntax with a directive to be consistent with using controller as with view and controller pairings.
-     *   https://github.com/johnpapa/angular-styleguide#style-y075
-     */
 
-    /**
-     * Unless your template is very small, it's typically better to break it apart into its own HTML file and load it with the templateUrl option.
-     *   https://docs.angularjs.org/guide/directive
-     */
     var directive = {
       restrict: 'EA',
       templateUrl: 'app/directives/side-bar/side-bar.directive.html',
       scope: {},
       controllerAs: 'vm',
-      controller: SideBarDirectiveController
+      controller: ["$scope", SideBarDirectiveController],
```

Comment:
Right, there are no injection. The controller will receive the same 4 arguments every time,
if you change the order, you will receive the same also in the same order